doc/dev/osd_internals/multi_object_write_transaction.rst

```diff
@@ -0,0 +1,113 @@
+==============================
+Multi-Object Write Transaction
+==============================
+
+Multi-object write transaction enables a group of write operations
+on multi-objects complete atomically.
+
+Description of the Process
+--------------------------
+
+The high level process is,
+
+  1. Define the following class,
+
+     class *CEPH_RADOS_API MultiObjectWriteOperation*
+
+     {
+
+       std::map<std::string, ObjectWriteOperation*> slaves;
```

Comment:
Is the std::string the slave object name?  I think it needs to be more than a std::string in order to accommodate the other info.. object_locator_t + object_t.